lib/control-connection.js

```diff
@@ -25,10 +25,11 @@ var schemaChangeTypes = {
 /**
  * Represents a connection used by the driver to receive events and to check the status of the cluster
  * @param {Object} options
+ * @param {ProfileManager} profileManager
  * @extends EventEmitter
  * @constructor
  */
-function ControlConnection(options) {
+function ControlConnection(options, profileManager) {
```

Comment:
`this.loadBalancingPolicy` doesn't pull from `profileManager`, although it looks like it isn't used anymore so could just be removed.